src/main/java/javaslang/collection/HashArrayMappedTrie.java

```diff
@@ -17,7 +17,7 @@
 /**
  * <a href="https://en.wikipedia.org/wiki/Hash_array_mapped_trie">Hash array mapped trie (HAMT)</a>.
  */
-abstract class HashArrayMappedTrie<K, V> implements Iterable<Tuple2<K, V>> {
+abstract class HashArrayMappedTrie<K, V> implements Iterable<Tuple2<K, V>>, Serializable {
 
```

Comment:
Ah - there is a unit test that checks, if all return types are as special as they can be.

Example:

Option.map returns Option but we know that None.map returns a None.

If we do not overwrite Option.map within None that way, it is put on the WHITELIST in the unit test.

You can copy the String 1:1 from the console and add a WHITELIST entry.